packages/gguf/src/types.ts

```diff
@@ -44,19 +47,7 @@ export enum GGUFValueType {
 	FLOAT64 = 12,
 }
 
-export const ARCHITECTURES = [
-	"llama",
-	"mpt",
-	"gptneox",
-	"gptj",
-	"gpt2",
-	"bloom",
-	"falcon",
-	"gemma",
-	"rwkv",
-	"whisper",
-] as const;
-
+const ARCHITECTURES = [...LLM_ARCHITECTURES, "rwkv", "whisper"];
```

Comment:
```suggestion
const ARCHITECTURES = [...LLM_ARCHITECTURES, "rwkv", "whisper"] as const;
```

I think you need to add this here otherwise the type inference is `string[]` – unless we're not using the same version of typescript?